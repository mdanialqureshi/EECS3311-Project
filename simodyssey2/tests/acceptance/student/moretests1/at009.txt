-- Test move preconditions 2.
--toggle_RNG_out
test(1,1,1,1,80)
pass
-- try to move into full sector
move(E)
-- land on planet
move(SE)
land

-- try to move into full sector while landed (test err msg priority)
move(N)

abort
