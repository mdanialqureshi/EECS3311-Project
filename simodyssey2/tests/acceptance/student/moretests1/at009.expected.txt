  state:0.0, ok
  Welcome! Try test(1,1,1,1,30)
->test(1,1,1,1,80)
  state:1.0, mode:test, ok
  Movement:none
  Sectors:
    [1,1]->[0,E],[1,P],-,-
    [1,2]->[2,P],[3,P],[-8,Y],-
    [1,3]->[4,P],[5,P],-,-
    [1,4]->[6,P],-,-,-
    [1,5]->[7,P],[-11,*],-,-
    [2,1]->[8,P],[9,P],-,-
    [2,2]->[10,P],[-6,Y],-,-
    [2,3]->[11,P],-,-,-
    [2,4]->-,-,-,-
    [2,5]->[12,P],[13,P],[14,P],[-4,*]
    [3,1]->[15,P],[16,P],-,-
    [3,2]->[17,P],[18,P],-,-
    [3,3]->[-1,O],-,-,-
    [3,4]->[19,P],[20,P],[21,P],[-3,W]
    [3,5]->[22,P],[23,P],[24,P],[-7,W]
    [4,1]->[25,P],-,-,-
    [4,2]->-,-,-,-
    [4,3]->[26,P],-,-,-
    [4,4]->[27,P],[28,P],[-10,W],-
    [4,5]->[29,P],-,-,-
    [5,1]->[30,P],[31,P],-,-
    [5,2]->[32,P],[-5,Y],-,-
    [5,3]->[33,P],[34,P],[-2,W],-
    [5,4]->[35,P],[-9,Y],-,-
    [5,5]->[36,P],[37,P],-,-
  Descriptions:
    [-11,*]->Luminosity:5
    [-10,W]->
    [-9,Y]->Luminosity:2
    [-8,Y]->Luminosity:2
    [-7,W]->
    [-6,Y]->Luminosity:2
    [-5,Y]->Luminosity:2
    [-4,*]->Luminosity:5
    [-3,W]->
    [-2,W]->
    [-1,O]->
    [0,E]->fuel:3/3, life:3/3, landed?:F
    [1,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [2,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [3,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [4,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [5,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [6,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [7,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [8,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [9,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [10,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [11,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [12,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [13,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [14,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [15,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [16,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [17,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [18,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [19,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [20,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [21,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [22,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [23,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [24,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [25,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [26,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [27,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [28,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [29,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [30,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [31,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [32,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [33,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [34,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [35,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [36,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [37,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
  Deaths This Turn:none
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    EP--   PPY-   PP--   P---   P*--   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    PP--   PY--   P---   ----   PPP*   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    PP--   PP--   O---   PPPW   PPPW   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    P---   ----   P---   PPW-   P---   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    PP--   PY--   PPW-   PY--   PP--   
->pass
  state:2.0, mode:test, ok
  Movement:
    [4,P]:[1,3,1]->[5,2,3]
    [5,P]:[1,3,2]->[1,2,4]
    [6,P]:[1,4,1]->[5,5,3]
    [18,P]:[3,2,2]->[4,1,2]
    [20,P]:[3,4,2]->[2,3,2]
    [21,P]:[3,4,3]->[3,3,2]
    [25,P]:[4,1,1]->[4,2,1]
    [27,P]:[4,4,1]->[4,5,2]
    [28,P]:[4,4,2]->[5,4,3]
  Sectors:
    [1,1]->[0,E],[1,P],-,-
    [1,2]->[2,P],[3,P],[-8,Y],[5,P]
    [1,3]->-,-,-,-
    [1,4]->-,-,-,-
    [1,5]->[7,P],[-11,*],-,-
    [2,1]->[8,P],[9,P],-,-
    [2,2]->[10,P],[-6,Y],-,-
    [2,3]->[11,P],[20,P],-,-
    [2,4]->-,-,-,-
    [2,5]->[12,P],[13,P],[14,P],[-4,*]
    [3,1]->[15,P],[16,P],-,-
    [3,2]->[17,P],-,-,-
    [3,3]->[-1,O],-,-,-
    [3,4]->[19,P],-,-,[-3,W]
    [3,5]->[22,P],[23,P],[24,P],[-7,W]
    [4,1]->-,[18,P],-,-
    [4,2]->[25,P],-,-,-
    [4,3]->[26,P],-,-,-
    [4,4]->-,-,[-10,W],-
    [4,5]->[29,P],[27,P],-,-
    [5,1]->[30,P],[31,P],-,-
    [5,2]->[32,P],[-5,Y],[4,P],-
    [5,3]->[33,P],[34,P],[-2,W],-
    [5,4]->[35,P],[-9,Y],[28,P],-
    [5,5]->[36,P],[37,P],[6,P],-
  Descriptions:
    [-11,*]->Luminosity:5
    [-10,W]->
    [-9,Y]->Luminosity:2
    [-8,Y]->Luminosity:2
    [-7,W]->
    [-6,Y]->Luminosity:2
    [-5,Y]->Luminosity:2
    [-4,*]->Luminosity:5
    [-3,W]->
    [-2,W]->
    [-1,O]->
    [0,E]->fuel:3/3, life:3/3, landed?:F
    [1,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [2,P]->attached?:T, support_life?:T, visited?:F, turns_left:N/A
    [3,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [4,P]->attached?:T, support_life?:F, visited?:F, turns_left:N/A
    [5,P]->attached?:T, support_life?:T, visited?:F, turns_left:N/A
    [6,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [7,P]->attached?:T, support_life?:F, visited?:F, turns_left:N/A
    [8,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [9,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [10,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [11,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [12,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [13,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [14,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [15,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [16,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [17,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [18,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [19,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [20,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [22,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [23,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [24,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [25,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [26,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [27,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [28,P]->attached?:T, support_life?:F, visited?:F, turns_left:N/A
    [29,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [30,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [31,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [32,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [33,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [34,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [35,P]->attached?:T, support_life?:F, visited?:F, turns_left:N/A
    [36,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [37,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
  Deaths This Turn:
    [21,P]->attached?:F, support_life?:F, visited?:F, turns_left:N/A,
      Planet got devoured by blackhole (id: -1) at Sector:3:3
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    EP--   PPYP   ----   ----   P*--   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    PP--   PY--   PP--   ----   PPP*   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    PP--   P---   O---   P--W   PPPW   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    -P--   P---   P---   --W-   PP--   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    PP--   PYP-   PPW-   PYP-   PPP-   
->move(E)
  state:2.1, mode:test, error
  Cannot transfer to new location as it is full.
->move(SE)
  state:3.0, mode:test, ok
  Movement:
    [0,E]:[1,1,1]->[2,2,3]
    [1,P]:[1,1,2]->[5,2,4]
    [6,P]:[5,5,3]->[4,4,1]
    [8,P]:[2,1,1]
    [11,P]:[2,3,1]->[3,2,2]
    [17,P]:[3,2,1]->[2,3,1]
    [19,P]:[3,4,1]->[2,4,1]
    [22,P]:[3,5,1]->[4,4,2]
    [23,P]:[3,5,2]->[4,4,4]
    [24,P]:[3,5,3]->[4,1,1]
    [26,P]:[4,3,1]
    [30,P]:[5,1,1]->[1,5,3]
    [31,P]:[5,1,2]
    [37,P]:[5,5,2]->[1,4,1]
  Sectors:
    [1,1]->-,-,-,-
    [1,2]->[2,P],[3,P],[-8,Y],[5,P]
    [1,3]->-,-,-,-
    [1,4]->[37,P],-,-,-
    [1,5]->[7,P],[-11,*],[30,P],-
    [2,1]->[8,P],[9,P],-,-
    [2,2]->[10,P],[-6,Y],[0,E],-
    [2,3]->[17,P],[20,P],-,-
    [2,4]->[19,P],-,-,-
    [2,5]->[12,P],[13,P],[14,P],[-4,*]
    [3,1]->[15,P],[16,P],-,-
    [3,2]->-,[11,P],-,-
    [3,3]->[-1,O],-,-,-
    [3,4]->-,-,-,[-3,W]
    [3,5]->-,-,-,[-7,W]
    [4,1]->[24,P],[18,P],-,-
    [4,2]->[25,P],-,-,-
    [4,3]->[26,P],-,-,-
    [4,4]->[6,P],[22,P],[-10,W],[23,P]
    [4,5]->[29,P],[27,P],-,-
    [5,1]->-,[31,P],-,-
    [5,2]->[32,P],[-5,Y],[4,P],[1,P]
    [5,3]->[33,P],[34,P],[-2,W],-
    [5,4]->[35,P],[-9,Y],[28,P],-
    [5,5]->[36,P],-,-,-
  Descriptions:
    [-11,*]->Luminosity:5
    [-10,W]->
    [-9,Y]->Luminosity:2
    [-8,Y]->Luminosity:2
    [-7,W]->
    [-6,Y]->Luminosity:2
    [-5,Y]->Luminosity:2
    [-4,*]->Luminosity:5
    [-3,W]->
    [-2,W]->
    [-1,O]->
    [0,E]->fuel:3/3, life:3/3, landed?:F
    [1,P]->attached?:T, support_life?:T, visited?:F, turns_left:N/A
    [2,P]->attached?:T, support_life?:T, visited?:F, turns_left:N/A
    [3,P]->attached?:T, support_life?:T, visited?:F, turns_left:N/A
    [4,P]->attached?:T, support_life?:F, visited?:F, turns_left:N/A
    [5,P]->attached?:T, support_life?:T, visited?:F, turns_left:N/A
    [6,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [7,P]->attached?:T, support_life?:F, visited?:F, turns_left:N/A
    [8,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [9,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [10,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [11,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [12,P]->attached?:T, support_life?:F, visited?:F, turns_left:N/A
    [13,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [14,P]->attached?:T, support_life?:F, visited?:F, turns_left:N/A
    [15,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [16,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [17,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [18,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [19,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [20,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [22,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [23,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [24,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [25,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [26,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [27,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [28,P]->attached?:T, support_life?:F, visited?:F, turns_left:N/A
    [29,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [30,P]->attached?:T, support_life?:F, visited?:F, turns_left:N/A
    [31,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [32,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [33,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [34,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [35,P]->attached?:T, support_life?:F, visited?:F, turns_left:N/A
    [36,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [37,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
  Deaths This Turn:none
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    ----   PPYP   ----   P---   P*P-   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    PP--   PYE-   PP--   P---   PPP*   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    PP--   -P--   O---   ---W   ---W   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    PP--   P---   P---   PPWP   PP--   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    -P--   PYPP   PPW-   PYP-   P---   
->land
  state:3.1, mode:test, error
  Negative on that request:no unvisited attached planet at Sector:2:2
->move(N)
  state:3.2, mode:test, error
  Cannot transfer to new location as it is full.
->abort
  state:3.3, ok
  Mission aborted. Try test(1,1,1,1,30)
